generals/integrations/pettingzoo_integration.py

```diff
@@ -88,7 +92,7 @@ def step(self, action):
 
         return observations, rewards, terminated, truncated, infos
 
-    def default_rewards(self, observations):
+    def default_rewards(self, observations: dict[str, Observation]) -> dict[str, int]:
```

Comment:
reward should be `float` (`int` is a coincidence now)

Reply:
done